proto/regen/ecocredit/basket/v1/query.proto

```diff
@@ -0,0 +1,97 @@
+syntax = "proto3";
+
+package regen.ecocredit.basket.v1;
+
+option go_package = "github.com/regen-network/regen-ledger/x/ecocredit/basket/v1";
```

Comment:
Are we needed to remove `/v1` now? @robert-zaremba

Reply:
yes

Reply:
fixed